### ISSUE TITLE
How to create eddy network from identification files

### ISSUE THREAD

**Issue by Ashwita2619** [AntSimi/py-eddy-tracker#152]
Hi Antione,

How to generate a network (network.nc or anti/cyclone_network.nc) file to use the different network manipulations
like shown here:
https://py-eddy-tracker.readthedocs.io/en/stable/python_module/16_network/pet_relative.html#sphx-glr-python-module-16-network-pet-relative-py
or here:
https://py-eddy-tracker.readthedocs.io/en/stable/python_module/16_network/pet_atlas.html#sphx-glr-python-module-16-network-pet-atlas-py

**Comment by AntSimi** [comment]
To create eddy network, you will use same identification files that you use for classical tracking. But tracking step is divide in 2 parts:

- One step to group eddy observations in group of observations with `EddyNetworkGroup`
- One step to divide group in several segment (consecutive observation without interaction) with `EddyNetworkBuildPath`


Group
=====

```bash
# This command will group anticyclonic observations of january 2019, each observation will be 
# compare with observation of the 5 next days and to be put in the same group overlap between 
# the 2 observations must be greater than 20%.
EddyNetworkGroup "Anticyclonic*_201901*" Anticyclonic_network_group.nc --window 5 --min-overlap .2
```
[This video](https://py-eddy-tracker.readthedocs.io/en/stable/python_module/16_network/pet_group_anim.html#anim) show group processing with a windows of 7 days

Please read documentation provide by :
```bash
EddyNetworkGroup -h
```

Segment
=======

```bash
# This command will split each group in several segment, here we allow 4 missing consecutiv detection with window parameter
EddyNetworkBuildPath Anticyclonic_network_group.nc Anticyclonic_network.nc -v INFO --window 5 --min-overlap .2
```

[This video](https://py-eddy-tracker.readthedocs.io/en/stable/python_module/16_network/pet_segmentation_anim.html#anim) show split processing for one group/network.

Please read documentation provide by :
```bash
EddyNetworkBuildPath -h
```

Analysis
========

For further analysis and manipulation take look at this [gallery](https://py-eddy-tracker.readthedocs.io/en/stable/python_module/index.html#network)

**Comment by Ashwita2619** [comment]
thank you…

**Comment by Ashwita2619** [comment]
I have generated the network files 
but, it has very strange time stored in its file:
>>> time_a.max()
0.2671412037037037
>>> time_a.min()
0.24192129629629627

Do you know why is it so?

**Comment by Ashwita2619** [comment]
To generate the network files: I have used the latest version available on github 
for the eddy identification v1.0 is used

The global attributes for both (network and identification) files are the same…

**Comment by Ashwita2619** [comment]
and unfortunately I can't find the documentation of:
EddyNetworkGroup and EddyNetworkBuildPath
which you have mentioned above…

**Comment by AntSimi** [comment]
Could you give version for one identification file that you use with this command or any other way which give you framework version:
```bash
ncdump -h your_idenfication_file.nc |grep framework_version
```

For software version please return result of:
```python
import py_eddy_tracker
print(py_eddy_tracker.__version__)
```

For help that you requested did you run in a terminal `command --help` ?

**Comment by Ashwita2619** [comment]
framework_version = "0+unknown"

and software version: 0+unknown

**Comment by AntSimi** [comment]
How did you install py eddy tracker python package?

**Comment by Ashwita2619** [comment]
I install py-eddy-tracker by running the following:
python setup.py install

and the for the different python packages I used conda and not pip to install

**Comment by AntSimi** [comment]
How did you get/copy source files on your computer?
I try to understand why you don't have version information.

**Comment by Ashwita2619** [comment]
I download the .zip file from the github
and aftering unziping the folder I installed the py-eddy-tracker

**Comment by AntSimi** [comment]
Ok, framework version is set only if you install by `pip install pyeddytracker` or if you get source code like that `git clone https://github.com/AntSimi/py-eddy-tracker`.

I think your time problem come from use of different version of software, it could be solve.
First could you check and share datatype of time in identification netcdf file?

**Comment by AntSimi** [comment]
When you talk about float did you share result of `ncdump` ?
Could you share result of this:
```bash
ncdump -h your_identification_file|grep time
```
Did you use  a modified version of py-eddy-tracker software to run eddy detection?

**Comment by Ashwita2619** [comment]
The result of the ncump is attached below (ncdump_identification_time.png)

I used the version realeased in May 2021 for the identification


<img width="873" alt="ncdump_identification_time" src="https://user-images.githubusercontent.com/53576269/180612233-27dbbb91-bb4a-45c1-a9c5-abc7b6861169.png">
)

**Comment by AntSimi** [comment]
Time has never been stored in float in original version of py-eddy-tracker, maybe you use a custom version like [this one from @mesharou](https://github.com/Mesharou/py-eddy-tracker/blob/master/src/py_eddy_tracker/__init__.py#L116)  to create your identification file.

Could you share few step of identification (maybe 10 files)?

**Comment by Ashwita2619** [comment]
Yes, I used the customed version to generate the identification files

I have attached the identification files of 10 days (20 files) where the detection is done every 12 hours.
[identification_files.zip](https://github.com/AntSimi/py-eddy-tracker/files/9180656/identification_files.zip)

**Comment by AntSimi** [comment]
This [branch](https://github.com/AntSimi/py-eddy-tracker/tree/unraw_network) must solve your problem when you use `EddyNetworkGroup ` add option `--unraw` because your time format are not directly compatible with raw mode.
To keep version information, my advice is to use git clone and not just download archive.
